ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,115 @@
+[[!meta title="High Performance Computing"]]
+
+**NOTE: We are in the process of trialing this service to users so that we can make the service as accommodating and secure as possible. This means that items concerning the service, including this documentation, are subject to change. We will do our best to keep everyone updated and notified of changes as they come.**
+
+## Introduction
+
+At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. We currently have one main HPC server, which handles all of our HPC jobs. However, have plans to expand the cluster to maximize the use of the resources at our disposal.
```

Comment:
"maximize the use of the resources at our disposal" is wordy...

How about just "maximize the resources at our disposal."

Reply:
> We currently have one main HPC server, which handles all our HPC jobs

is also doubly redundant: if we only have one HPC server, it is necessarily our main HPC server, and also the one that handles all our HPC jobs.